src/components/snackbar/examples/snackbar-positioning.scss

```diff
@@ -1,5 +1,5 @@
 limel-snackbar {
-    --snackbar-top: 0;
+    --snackbar-top: 0.5rem;
```

Comment:
this means we should also update the code in the webclient, as we're rendering the snackbars on the top-right location

Reply:
I made some changes in the styles. So we now don't have to update the web-client this anymore.